pkg/plan/recipe/install_plans.go

```diff
@@ -30,6 +31,19 @@ func BuildBasePlan(pkgType resource.PkgType) plan.Resource {
 		// Device Mapper
 		b.AddResource("install:device-mapper-persistent-data", &resource.RPM{Name: "device-mapper-persistent-data"})
 		b.AddResource("install:lvm2", &resource.RPM{Name: "lvm2"})
+
+	case resource.PkgTypeRHEL:
```

Comment:
I'd prefer to see resource.PkgTypeTHEL adde to the resource.PkgTypeRPM case and add an if for adding the install of container-selinux